control-plane/agents/src/bin/core/main.rs

```diff
@@ -162,6 +162,18 @@ pub(crate) struct CliArgs {
     /// to configure cluster size per-pool.
     #[clap(long)]
     pool_cluster_size: Option<u32>,
+
+    /// Running in simulation mode.
+    #[clap(long, env = "SIMULATION", requires = "bundle")]
+    simulation: bool,
```

Comment:
Or maybe hide this?

Reply:
No one is running agent-core directly like the plugin so I think it's probably fine to show these, but sure I'll hide it then

Reply:
Upto you, it was more of a nit.